luigi/contrib/ecs.py

```diff
@@ -182,6 +182,11 @@ def run(self):
         response = client.run_task(taskDefinition=self.task_def_arn,
                                    overrides=overrides,
                                    cluster=self.cluster)
+
+        if response['failures']:
+            raise Exception(", ".join(["fail to run task {0} reason: {1}".format(failure['arn'], failure['reason'])
```

Comment:
will `failure` always include `arn` and `reason` in its dictionary?

If so, :+1:

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task 
Assuming I'm reading the AWS Boto3 documentation right, yes I believe so. Failures is a list of 0 or more elements. If elements exist, they are of form

```{
            'arn': 'string',
            'reason': 'string'
        },```